lib/SDEProblemLibrary/Project.toml

```diff
@@ -10,7 +10,7 @@ RuntimeGeneratedFunctions = "7e49a35a-f44a-4d26-94aa-eba1b4ca6b47"
 
 [compat]
 Aqua = "0.5"
-Catalyst = "11, 12"
+Catalyst = "13"
```

Comment:
```suggestion
Catalyst = "11, 12, 13"
```

Reply:
no because the new parameter form isn't supported